app/services/api/v1/message_service.rb

```diff
@@ -11,7 +11,15 @@ def create
         conversation = Conversation.find(@conversation_id)
         raise InvalidConversationError unless conversation.user1_id == @current_user.id || conversation.user2_id == @current_user.id
 
-        Message.create!(content: @message_params[:content], user: @current_user, conversation_id: @conversation_id)
+        @message = Message.create!(content: @message_params[:content], user: @current_user, conversation_id: @conversation_id)
+        notify if @message.persisted?
+        @message
+      end
+
+      def notify
+        conversation = Conversation.find(@conversation_id)
```

Comment:
Maybe you can pass the conversation as a parameter so you don't have to find it again.
Also, I think this method could be a private method